en/messages.json

```
@@ -223,6 +223,9 @@
   "enableSyncDescription": {
     "message": "This setting allows you to sync your Containers and site assignments across devices."
   },
+  "syncExcludeDescription": {
+    "message": "Exclude containers based on their name from sync by specifying a case insensitive regular expression."
```

The word order sounds a bit strange to me (moving `from sync` after the object seems better). Thoughts about this?

```suggestion
    "message": "Exclude containers from sync when their name matches the specified case insensitive regular expression."
```